### PR TITLE
Chore/override jackson version

### PR DESCRIPTION
**Pull request by JosephKelly**
A bit of a shortcut for working through the nested dependencies

### CODE REVIEW

File: tcs-api/pom.xml [hunk 1]
```diff
@@ -21,12 +21,17 @@
...
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.9.10.7</version>
+      <version>2.13.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+      <version>2.13.0</version>
     </dependency>
```
Comment (Judge40) [inline tcs-api/pom.xml:35]: Is this duplicate meant to be here?

**Review by jayanta2018 — DISMISSED** [review]

File: tcs-api/pom.xml (hunk 1 above)
Comment (JosephKelly) [reply tcs-api/pom.xml:35]: Nope. Thanks. :sleeping:

**Comment by jayanta2018** [comment]
Tests are failing. Not sure, why

**Comment by JosephKelly** [comment]
> Tests are failing. Not sure, why

There were local builds with version numbers giving me a false pass.

**Comment by JosephKelly** [comment]
Issue with third-party github action when base of branch is behind PR target.  Superseded by a rebased #771.

**Pull request closed without merging**